src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -1150,7 +1150,7 @@ export class CalciteSlider implements LabelableComponent, FormComponent {
     const labelOffset = labelFontSize / 2;
 
     if (labelTransformedOverlap > 0) {
-      hyphenLabel.classList.add("hyphen");
+      hyphenLabel.classList.add("hyphen", "handle__label--flex");
```

Comment:
Does this class need to be removed at some point in runtime?

Reply:
It can be removed considering when hyphenation is not required.